src/Resources/config/core.xml

```diff
@@ -71,5 +71,11 @@
         <service id="sonata.admin.twig.global" class="Sonata\AdminBundle\Twig\GlobalVariables" public="true">
             <argument type="service" id="sonata.admin.pool"/>
         </service>
+        <!-- templates -->
+        <service id="sonata.admin.template_registry" class="Sonata\AdminBundle\Admin\TemplateRegistry" public="false">
```

Comment:
Why does the pool need a template registry by the way? Should this be rename to indicate what particularity this template registry has?

Reply:
Through the `Pool::getTemplate()` method, general templates are loaded (the ones that are not specific for an admin), like layout and ajax.

I could use `"sonata.admin.general_template_registry"` instead. I would prefer not to use "pool" anywhere.

Reply:
I think ajax is just another kind of layout, so maybe have that `layout` word in there?

Reply:
The templates `add_block`, `user_block` and `tab_menu_template` are also accessed this way.

Reply:
Ok, how about `global` instead? `general` feels a bit awkward?

Reply:
Oh, that's better. Thanks!